### ISSUE TITLE
Move taxon from one parent to another in the Tree: error 400

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/backend#1019]
Frontend version: 41cae4e June 8, 2021 7:28 AMBackend version: 654bc36 June 2, 2021 5:54 AM

For attention of @thomasstjerne & @mdoering 

On ITIS request, I have tried to move infraorder Astacidea from suborder NotAssigned to [sister] suborder Pleocyemata. (Assembly - ModifyTree mode). 

As result, I have got error message with code 400:

![image](https://user-images.githubusercontent.com/28574252/121575926-88885400-c9ed-11eb-8cc5-43ed9029ae2c.png)

Console says:

Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/” because the scheme does not match. data.catalogueoflife.org
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/css/main.81a46273.css” because the scheme does not match. main.81a46273.css
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/css/main.81a46273.css” because the scheme does not match. main.81a46273.css
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/css/main.81a46273.css” because the scheme does not match. main.81a46273.css
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. data.catalogueoflife.org
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/js/main.67bfeb5d.js” because the scheme does not match. main.67bfeb5d.js
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/js/main.67bfeb5d.js” because the scheme does not match. main.67bfeb5d.js
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/static/js/main.67bfeb5d.js” because the scheme does not match. main.67bfeb5d.js
false index.js:108:22
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/terms.json” because the scheme does not match. terms.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/terms.json” because the scheme does not match. terms.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/terms.json” because the scheme does not match. terms.json
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/favicon.ico” because the scheme does not match. favicon.ico
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/favicon.ico” because the scheme does not match. favicon.ico
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/favicon.ico” because the scheme does not match. favicon.ico
false index.js:108:22
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
false index.js:108:22
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
false index.js:108:22
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning; see https://developer.mozilla.org/docs/Mozilla/Performance/ScrollLinkedEffects for further details and to join the discussion on related tools and features! classification
false index.js:108:22
Cookie “__utmc” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmt” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json
Cookie “__utmb” will be soon treated as cross-site cookie against “https://data.catalogueoflife.org/gitVersion.json” because the scheme does not match. gitVersion.json

​

**Comment by yroskov** [comment]
Well, succeeded to move it after deletion of the sector, re-establishing the sector and re-sync.

**Comment by yroskov** [comment]
If taxa cannot be moved inside the tree with attached sectors, it would be nice to add an appropriate warning in the interface.

However:
Move taxa from one parent to another with attached sectors may help to modify the tree without big amount of extra work for taxa with multiple sectors.

**Comment by mdoering** [comment]
Yes, you are not allowed to modify the tree inside a sector. Only taxa without a source.
@thomasstjerne the UI should warn users and not allow them to do so.
